src/decision_process/examples.md

```diff
@@ -0,0 +1,297 @@
+
+> Dear @rust-lang/team, @Alan has restarted the clock!
+
+The status now looks like this:
+
+> Hello! @Alan has proposed to merge this. This is a **reversible decision**,
+> which means that it will be affirmed once the "final comment period" of 10
+> days have passed, unless a team member pauses or suspends the decision.
+>
+> | Team member | State |
+> | --- | --- |
+> | @Alan | **merge** |
+> | @Barbara | ~~merge~~ |
+> | @Grace | |
+> | @Niklaus | ~~hold~~ |
+
+Barbara can use `@rustbot merge` to re-affirm her **merge** status, and Niklaus
+can use `@rustbot merge` to set his own status to **merge** since he agrees
+with the resolution of his concern.
+
+## Authoring an RFC continued (Overriding a concern)
+
+At this point, Grace has a concern, and explains that concern in detail:
+
+> @rustbot hold
+>
+> I've thought about this a lot, and I don't think we should do this. Now that
+> I see the syntax used in practice, I feel like if we do this it'll have an
+> adverse effect on the ecosystem...
+
+The status now looks like this:
+
+> Hello! @Alan has proposed to merge this. This is a **reversible decision**,
+> which means that it will be affirmed once the "final comment period" of 10
+> days have passed, unless a team member pauses or suspends the decision.
+>
+> | Team member | State |
+> | --- | --- |
+> | @Alan | **merge** |
+> | @Barbara | ~~merge~~ **merge** |
+> | @Grace | **hold** |
+> | @Niklaus | ~~hold~~ **merge** |
+
+Niklaus reads this message. He feels he understands the concern well, and
+agrees that this point hasn't yet been considered:
+
+> @rustbot hold
+>
+> I agree. I think we should take more time to evaluate alternative syntaxes.
+> What about...
+
+Over the course of a few subsequent meetings and side conversations, Grace and
+other team members discuss the concern further; the initiative owner also
+considers the concern, and raises it with others working on the initiative.
+
+The owner of the initiative updates the RFC to include a discussion of a couple
+of alternative syntax proposals. The owner recommends a slightly modified
+version of the originally proposed syntax, and outlines criteria that they feel
+the syntax should meet in order to support the use case.
+
+Grace agrees that her concern has been understood, but does not agree with the
+proposed syntax. Grace feels the new proposal is an improvement, but her
+concern remains.
+
+Niklaus feels that the team has understood Grace's concern, and furthermore,
+that the updated proposal addresses Grace's concern:
+
+> @rustbot merge
+>
+> I appreciate the potential impact this may have on the ecosystem. However, I
+> feel that as now described in section XYZ of the RFC, the value of A
+> outweighs the risk of B, and we've now mitigated the potential risk by...
+
+(Notice that while Niklaus feels that the team has understood Grace's concern,
+he does not use "We" in his response, and does not imply that his summary
```

Comment:
Well, Niklaus *did* use "we", namely in:

> and we've now mitigated the potential risk by...

Am I nitpicking? I don't know. I think the detail the text is pointing out here is super-important, in terms of emphasizing clarity in attributing which opinions belong to individuals, and which belong to the team as a whole.

Maybe easiest to just rewrite Niklaus's comment to eschew "we" 100%?

E.g.

"outweighs the risk of B, and it now mitigates the potential risk by..."

Reply:
I don't think you're nitpicking at all. I reworded the guidance here to "does not speak for the entire team" rather than tabooing the word "we". However, I also rewrote the sentence to change "we've" to an "I think" phrasing instead.